workers/pipeline/create_pipeline.go

```diff
@@ -139,6 +141,16 @@ func CreatePipeline(p *gaia.CreatePipeline) {
 		return
 	}
 
+	nsUUID := uuid.NewV4()
```

Comment:
In my opinion, this should be moved to https://github.com/gaia-pipeline/gaia/pull/158/files#diff-3ea375c9ea9839dd14147d6613685a41R127

It's risky to save the pipeline object after "CopyBinary" because that introduces a race condition with the ticker.

Reply:
Woa. Nice catch! Thanks.

Reply:
@michelvocks Where does your link point to? 🤔

Reply:
@michelvocks Like that? :)